### DISCUSSION TITLE
PK-sim sensitivity analysis blank

### DISCUSSION THREAD

**Discussion by learnCode-cpu** [Open-Systems-Pharmacology/Forum#1226] [Questions & Problems]
Hi, I have created a simulation in PK-Sim. I want to carry out sensitivity analysis. However, when I try to do it, the parameters section on the sensitivity analysis page is blank; actually, nothing can be selected. I was looking at the documentation which states "Mark a simulation for the Sensitivity Analysis in the Simulation Explorer" - how can I mark a simulation for SA? I don't see that option. 

I think I am missing something basic in setting up the simulation. I was wondering if anyone help me with this issue. 

Thanks, 
John

**IMPORTANT: Issues are now obsolete**
Starting May 7th, 2021, the OSP issues will not be used anymore for discussions. Instead, the Discussions feature from Github will be used

Please see the announcement here:
https://github.com/Open-Systems-Pharmacology/Forum/issues/894

To start a discussion, simply go to 
https://github.com/Open-Systems-Pharmacology/Forum/discussions and click on New Discussion

**Comment by msevestre** [comment]
Hi @learnCode-cpu ,

>However, when I try to do it, the parameters section on the sensitivity analysis page is blank; actually, nothing can be selected. 

Can you please show us a screenshot? It is very hard to diagnose what your problem might be if we do not have more information

**Reply by learnCode-cpu** [reply]
I have shared the screenshot below. 

![sensitivity analysis](https://user-images.githubusercontent.com/107363086/173257772-5f16d0e3-f433-416b-b23a-2b1fca4dca1e.jpg)

**Comment by msevestre** [comment]
It's because sensitivity only works with individual simulation. The only simulation in your project seems to be a population Simulation.

So simply create a simulation with the same building blocks, swapping out the population for the individual, and you'll be good to go.

**Reply by learnCode-cpu** [reply]
Thanks - let me give that a try.

**Reply by learnCode-cpu** [reply]
Okay so I started with an individual. I have run into a couple of issues. I am looking at a specific transporter MDR1.

1. When I create an individual, it does not let me set the expression value of MDR1 as it is already in the project. If I create another individual, same thing. I am not sure why I cannot set the values independently. 
2. 
3. In the simulation panel, under parameters, there is a section called 'enzymes, transporters..' under which I see expression values for MDR1 (screenshot below). I can change these values. However, these changes do not show up in the individual. So where are these values being used? Are these being used or are the values for the individual being used when the simulation is solved? 
![image](https://user-images.githubusercontent.com/107363086/173261450-74abcd6e-a5ed-4f43-b4f2-06b90140df96.png)

In the big picture, I am trying to study how the expression of a transporter would affect absorption in the colon. I wasn't sure which parameter to look at as the outcome, so I am looking at everything for the time being. So far, I don't see expression of the transporter appear as a significant parameter in the sensitivity analysis but I am sure something is not correct in the way I have set up things. 

Thank you, 
John

**Comment by msevestre** [comment]
Expression values are defined in the building block called "Expression profile". One individual uses one or more expression profiles. So if you want to have multiple profiles for MDR1, you need to create multiple expression profiles for this transporter and use the corresponding profile in your individual.

When you change a value in the simulation, it it saved in the simulation. As you can see the simulation turns red, indicating that there is at least one difference between the building block in the project and the values in the simulation. To update your individual building blocks, right click on the individual in the simulation, with the red icon, and click Commit. This will commit your changes to the project individual. 

I would strongly suggest that you spend some time reading the documentation. The section on building blocks will help you get a better grasp as to how all the pieces are fitting together. 

Hope that helps,
